LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+BSD 3-clause license
```

Comment:
This should just be a file called LICENSE, and should have an APACHE 2 license instead of a BSD 3 one. This should be the license for the actual recipe and be the same one used in all of the other Open-CE recipes.